doctoc.js

```diff
@@ -114,3 +114,8 @@ for (var i = 0; i < argv._.length; i++) {
 
   console.log('\nEverything is OK.');
 }
+
+module.exports.transform=function(content, mode, maxHeaderLevel, title, notitle, entryPrefix){
+  return transform(content, mode, maxHeaderLevel, title, notitle, entryPrefix);
+};
```

Comment:
You could write this more simply as

```js
module.exports.transform = transform
```

Reply:
yes i agree. sorry i'm new to nodejs i just needed the API

Reply:
No problem, just go ahead and make that change and I'll merge it in. :)

Reply:
done!